dough-skins/src/main/java/io/github/bakedlibs/dough/skins/nms/PlayerHeadAdapter.java

```diff
@@ -26,9 +26,12 @@
         try {
             MinecraftVersion version = MinecraftVersion.get();
 
-            if (version.isAtLeast(1, 17)) {
-                // 1.17+ mappings
-                return new PlayerHeadAdapterAfter17();
+            if (version.getMajorVersion() == 1 && version.getMinorVersion() == 17) {
+                // 1.17 mappings
+                return new PlayerHeadAdapter17();
+            } else if (version.isAtLeast(1, 18)) {
+                // 1.18 mappings
+                return new PlayerHeadAdapter18();
```

Comment:
How come you're doing this over 
```suggestion
            if (version.isAtLeast(1, 18)) {
                // 1.18 mappings
                return new PlayerHeadAdapter17();
            } else if (version.isAtLeast(1, 17)) {
                // 1.17 mappings
                return new PlayerHeadAdapter18();
```